pandas/core/resample.py

```diff
@@ -239,7 +239,9 @@ def aggregate(self, arg, *args, **kwargs):
         self._set_binner()
         result, how = self._aggregate(arg, *args, **kwargs)
         if result is None:
+            grouper = None
             result = self._groupby_and_aggregate(arg,
+                                                 grouper,
```

Comment:
Ideally we would pass both ``how`` and ``grouper`` as kwargs, but you can only do that in PY3, ok
add ad ``how=arg`` and pass ``how`` to make this more clear then (positionally as the first arg, just where arg is now)

Reply:
and actually rename ``arg`` to ``func``, which is what it is called elsewhere